src/main/java/io/kestra/plugin/fs/ftp/Uploads.java

```diff
@@ -0,0 +1,61 @@
+package io.kestra.plugin.fs.ftp;
+
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.runners.RunContext;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.apache.commons.vfs2.FileSystemOptions;
+
+import java.io.IOException;
+import java.net.Proxy;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+        title = "Upload files to a FTP server's directory"
+)
+@Plugin(
+        examples = {
+                @Example(
+                        code = {
+                                "host: localhost",
+                                "port: 21",
+                                "username: foo",
+                                "password: pass",
+                                "from:",
+                                "  - \"{{ outputs.taskid1.uri }}\"",
+                                "  - \"{{ outputs.taskid2.uri }}\"",
+                                "to: \"/upload/dir2\"",
+                        }
+                )
+        }
+)
+public class Uploads extends io.kestra.plugin.fs.vfs.Uploads implements FtpInterface {
+    protected String proxyHost;
+    protected String proxyPort;
+    protected Proxy.Type proxyType;
+    @Builder.Default
+    protected Boolean rootDir = true;
+    @Builder.Default
+    protected String port = "21";
+    @Builder.Default
+    protected Boolean passiveMode = true;
+    @Builder.Default
+    protected Boolean remoteIpVerification = true;
```

Comment:
I think all those can be private as we use builders all over the place.
The same apply to all other tasks.